controlpanel/frontend/jinja2/parameter-create.html

```diff
@@ -0,0 +1,108 @@
+{% from "input/macro.html" import govukInput %}
+{% from "fieldset/macro.html" import govukFieldset %}
+{% from "radios/macro.html" import govukRadios %}
+
+{% extends "base.html" %}
+
+{% set page_title = "Create a secret parameter for your webapp or airflow task" %}
+{% set legend -%}
+  Create a secret parameter for your webapp or airflow task
+{%- endset %}
+
+{% set page_name = "parameters" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">{{ page_title }}</h1>
+
+  {% if form.error %}
+    <div class="error">{{ form.error }}</div>
+  {% endif %}
+
+  <form action="{{ url('create-parameter') }}" method="post">
+    {{ csrf_input }}
+    {% call govukFieldset({ "legend": { "html": legend|safe } }) %}
+      {{
+        govukInput({
+          "label": {
+            "text": "Key",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": '60 chars max, only letters, numbers and underscores'
+          },
+          "name": "key",
+          "attributes": {
+            "pattern": "[a-zA-Z0-9_]{1,60}",
+            "maxlength": "60",
+          },
+          "value": form.key.value(),
+          "errorMessage": { "html": form.key.errors|join("<br>") } if form.key.errors else {}
+        })
+      }}
+      {{
+        govukInput({
+          "label": {
+            "text": "Value",
+            "classes": "govuk-label--m",
+          },
+          "classes": "govuk-!-width-two-thirds",
+          "hint": {
+            "text": 'String. Any characters'
```

Comment:
Not sure this hint is very friendly

Reply:
Maybe something like "Value for the parameter. All characters allowed"? (or something like that)

Reply:
If there's no restriction, I think you can omit the "Any characters" or "All characters allowed".

Reply:
Removed